CHANGELOG.md

```diff
@@ -0,0 +1,1298 @@
+# Changelog
+This is the changelog for *casual java* and all changes are listed in this document.
+
+
+## [3.3.1] - 2025-01-13
+- Feature/3.2/unmarshall all null values (#143)
```

Comment:
Would be really nice if issue refs were hyperlinked like in the main casual repo. It seems to generate lines like `- xatmi: build-server uses [0,1] as values for service visibility ([#506](https://github.com/casualcore/casual/issues/506))`

Reply:
That should be easy enough to add, would be by convention then.
If you have a #[num*] then we'll treat that as an issue and can insert the link.

Reply:
I believe - need to test this myself to check - but this should already work with just the # and a number. See the second link below.
So if you view the markdown file on github ui it will work is my understanding from the documentation.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#referencing-issues-and-pull-requests

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls

Reply:
https://github.com/casualcore/casual-java/blob/feature/changelog/CHANGELOG.md

it seems that does not work?
See #143, #141

Reply:
Fixed, handles 0 - n issue numbers that are automagically translated to md links for the issue(s)